static/js/brand-store/Settings/Settings.tsx

```diff
@@ -83,8 +111,8 @@ function Settings() {
     setIsPrivateStore(!isPrivateStore);
   };
 
-  const handleRadioButtonChange = (e) => {
-    setManualReviewPolicy(e.target.value);
+  const handleRadioButtonChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setManualReviewPolicy((e.target as HTMLInputElement).value);
```

Comment:
Typically when you have to use `as` it's a point to stop and take a second glance....it shouldn't be required in the majority of cases.

In this case I think you want `e.currentTarget` instead.